internal/nodes/avi_model_nodes.go

```diff
@@ -1162,6 +1167,7 @@ func (v *AviPoolNode) CalculateCheckSum() {
 		v.SslProfileRef,
 		v.PriorityLabel,
 		utils.Stringify(nodeNetworkMap),
+		utils.Stringify(v.ServiceMetadata.NamespaceServiceName),
```

Comment:
+1 why would we want the ServiceMetadata to affect the checksum? The field by nature is used not as a typed one, it's a free flowing dict.

Reply:
I think we would need a PUT on the  pools to get in the service metadata in the **existing** pool configurations. This applies for only the gateway api related pools. Since this fixes a problem with status update on Services, the Pool PUT is required. I can limit the checksum calculation to include this only when `len(v.ServiceMetadata.NamespaceServiceName)>0`.